tests/detect-bytemath-mult-04/README.md

```diff
@@ -0,0 +1,11 @@
+Description
+===========
+Test byte_math multiplication operator
+
+PCAP
+====
+PCAP comes from the suricata verify test[detect-bytemath-01]
```

Comment:
You can reuse it with test.yaml pointing to the pcap
cf `git grep 'pcap: ../sip-method/sip.pcap'`

Reply:
Thanks -- will update.

Reply:
Done

Reply:
I think you need to remove the input.pcap file, and write in test.yaml `pcap: ../detect-bytemath-01/input.pcap`

Reply:
Thanks ... Will do.